templates/skeleton/daml.yaml.template

```diff
@@ -12,3 +12,18 @@ dependencies:
   - daml-script
 sandbox-options:
   - --wall-clock-time
+
+# https://discuss.daml.com/t/making-the-most-out-of-daml-compiler-warnings/739
+build-options: [
+  # make compilation fail if there is a warning
+  #"--ghc-option", "-Werror",
+
+  # turn on all warnings; this may be too verbose
+  #"--ghc-option", "-Wall",
```

Comment:
This has false positiives by default so I wouldn’t include this here.

Reply:
removed

Reply:
I don’t think you did?

Reply:
I had, but GH was broken. Seems the UI caught up now.